go/chat/unfurl/unfurler_test.go

```diff
@@ -115,6 +115,7 @@ func TestUnfurler(t *testing.T) {
 	}
 	require.NoError(t, settings.WhitelistAdd(context.TODO(), uid, "0.1"))
 
+	unfurler.UnfurlAndSend(context.TODO(), uid, convID, fromMsg)
```

Comment:
throw is in a 5-10x loop? i'm sure CI would have fun with creating interesting execution flows